ci-operator/step-registry/cucushift/installer/check/nutanix/network/cucushift-installer-check-nutanix-network-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: cucushift-installer-check-nutanix-network
```

Comment:
how about renaming it to `cucushift-installer-check-nutanix-infra`

Reply:
Fixed, thanks.